cumulus_library/studies/core/manifest.toml

```diff
@@ -4,22 +4,18 @@ study_prefix = "core"
 file_names = [
     "builder_prereq_tables.py",
     "builder_condition.py",
-    "builder_encounter_coding.py",
-    "builder_core_medication.py",
-    "builder_patient_extension.py",
-
+    "builder_patient.py",
+    "builder_encounter.py",
+    "builder_documentreference.py",
+    "builder_medication.py",
+    "builder_medicationrequest.py",
+    "builder_observation.py"
 ]
 
 [sql_config]
 file_names = [
```

Comment:
I left three bare sql files in here since they didn't touch the FHIR tables at all, but I might? move them to be loaded from the various builders, the way the prereq builder works.